integration-cli/docker_cli_stats_test.go

```diff
@@ -97,10 +97,7 @@ func (s *DockerSuite) TestStatsAllNoStream(c *check.C) {
 
 func (s *DockerSuite) TestStatsAllNewContainersAdded(c *check.C) {
 	// Windows does not support stats
-	// TODO: remove SameHostDaemon
```

Comment:
Removed so I can run win2lin against this more. It may very well resolve this issue.